pkg/registry/event/event_test.go

```diff
@@ -0,0 +1,37 @@
+package event_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	testclient "k8s.io/client-go/kubernetes/fake"
+
+	modeljobsv1alpha1 "github.com/kleveross/klever-model-registry/pkg/apis/modeljob/v1alpha1"
+	modeljobfake "github.com/kleveross/klever-model-registry/pkg/clientset/clientset/versioned/fake"
+	"github.com/kleveross/klever-model-registry/pkg/registry/client"
+	"github.com/kleveross/klever-model-registry/pkg/registry/event"
+	"github.com/kleveross/klever-model-registry/pkg/registry/modeljob"
+)
+
+var _ = Describe("Event", func() {
+	client.KubeMainClient = testclient.NewSimpleClientset()
```

Comment:
Prefer k8sfake here since we are using modeljobfake below.

Reply:
comment are addressed, PTAL

Reply:
I mean 
```suggestion
	client.KubeMainClient = k8sfake.NewSimpleClientset()
```

Reply:
oh, add and push is error, I should check it, now PTAL